ModList/UI/Markdig/UnityRenderer.cs

```diff
@@ -389,92 +419,91 @@ private IEnumerable<RectTransform> RenderInline(Inline inline, float fontSize, b
+            LinkInline link => RenderLinkInlineToText(link, builder),
+            HtmlInline tag => RenderHtmlInlineToText(tag, builder),
+            HtmlEntityInline entity => RenderHtmlEntityToText(entity, builder),
+
+            ContainerInline container => RenderContainerInlineToText(container, builder),
+            _ => throw new NotImplementedException($"Unknown inline type {inline.GetType()}")
+        };
 
         private StringBuilder RenderContainerInlineToText(ContainerInline container, StringBuilder builder)
         {
-            foreach (var inline in container)
-                builder = RenderInlineToText(inline, builder);
-            return builder;
+            return container.Aggregate(builder, (current, inline) => RenderInlineToText(inline, current));
         }
 
-        private StringBuilder RenderLiteralToText(LiteralInline lit, StringBuilder builder)
-            => builder.Append("<noparse>").AppendSlice(lit.Content).Append("</noparse>");
+        private StringBuilder RenderLiteralToText(LiteralInline lit, StringBuilder builder) => builder.Append("<noparse>").AppendSlice(lit.Content).Append("</noparse>");
 
         private StringBuilder RenderLineBreakInlineToText(LineBreakInline lb, StringBuilder builder)
             => builder.Append(lb.IsHard ? "\n" : " ");
 
-        private const string CodeRegionLinkIdStart = "__CodeInline__";
-        private int codeRegionLinkPostfix = 0;
+        private const string CodeRegionLinkIDStart = "__CodeInline__";
+        private int _codeRegionLinkPostfix = 0;
+
         private StringBuilder RenderCodeInlineToText(CodeInline code, StringBuilder builder)
             => builder.Append(CodeFont == null ? "" : $"<font=\"{CodeFont.name}\">")
-                      .Append("<size=80%>")
-                      .Append($"<link=\"{CodeRegionLinkIdStart}{codeRegionLinkPostfix++}\">")
-                      .Append(InlineCodePaddingText)
-                      .Append("<noparse>")
-                      .Append(code.Content)
-                      .Append("</noparse>")
-                      .Append(InlineCodePaddingText)
-                      .Append("</link></size>")
-                      .Append(CodeFont == null ? "" : "</font>");
-
-        private StringBuilder RenderHtmlInlineToText(HtmlInline tag, StringBuilder builder)
-            => builder.Append(tag.Tag);
-
-        private StringBuilder RenderHtmlEntityToText(HtmlEntityInline entity, StringBuilder builder)
-            => builder.AppendSlice(entity.Transcoded);
+                .Append("<size=80%>")
+                .Append($"<link=\"{CodeRegionLinkIDStart}{_codeRegionLinkPostfix++}\">")
+                .Append(InlineCodePaddingText)
+                .Append("<noparse>")
+                .Append(code.Content)
+                .Append("</noparse>")
+                .Append(InlineCodePaddingText)
+                .Append("</link></size>")
+                .Append(CodeFont == null ? "" : "</font>");
```

Comment:
I typically like having all of the chained method calls aligned with the first, though I don't care about that quite as much as some of the other things I've mentioned.